src/planscape/planning/permission.py

```diff
@@ -0,0 +1,14 @@
+from rest_framework.permissions import BasePermission
+
+
+class UserPermission(BasePermission):
+    def has_permission(self, request, view):
+        return self.is_authenticated(request)
```

Comment:
This is still WIP, but for this PR I was only going to implement list(), and in a list view, [apparently DRF doesn't do object-level permission checks ](https://www.django-rest-framework.org/api-guide/permissions/#limitations-of-object-level-permissions). 
Since our `CheckPermissionMixin` methods are only for objects, I didn't think there's anything we could add here. Our existing list views also just rely on queryset filtering to ensure that we only get objects that the user can see, using `get_for_user()` with so that's what I'm relying on here, as well.